README.md

```diff
@@ -5,3 +5,8 @@ NUSSU commIT Duty Website, iteration 4
 
 ## Schema Diagram
 ![Schema diagram](schema.jpg)
+
+## Entity-Relationship Diagram
+To generate the current Entity-Relationship Diagram, install `graphviz` (`sudo apt install graphfiz` or `brew install graphfiz`),
```

Comment:
Put this at the bottom of the diagram in a new subheading. `### Generate ERD` maybe?